src/main/java/be/syntra/devshop/DevshopFront/models/dtos/CartDisplayProductDto.java

```diff
@@ -0,0 +1,15 @@
+package be.syntra.devshop.DevshopFront.models.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Builder
+@AllArgsConstructor
+@Getter
+@Setter
+public class CartDisplayProductDto {
```

Comment:
a CartDisplayDto is a dto that displays the Cart. A CartDisplayProductDto sounds to like a ProductDto that displays a Cart. I'm rambling on about naming, I know. But doesn't CartProductDisplayDto read like a dto that displays a CartProduct?

Reply:
it's kind of a no win situation here, and i totally get your concern, there's cases for CartDisplayProductDto , CartProductDisplayDto and even CartDisplayDtoProduct... BUT i noticed CartDisplayProductDto is the same as CartProductDto, so i replaced it with the latter.

Reply:
My major concern is that it seems awfully convoluted to add this many dto's and abstractions just to replace one field on an entity. I'm sure that something like this wouldn't take place in the workfield. However, since we had no prior knowledge of how to properly utilize the concept, and we didn't take this into account when analyzing the project prior to our work... This has become a neat solution. So kudo's for refactoring it all.